### PR TITLE
update to 2.6.3

### PR DESCRIPTION
**Pull request by boldorider4**
* [PKG-3539]
* [Upstream requirements](https://github.com/scikit-learn-contrib/category_encoders/blob/2.6.3/setup.py)

[PKG-3539]: https://anaconda.atlassian.net/browse/PKG-3539?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,33 @@
...
-    - m2-patch    # [win]
   host:
     - python
     - pip
     - wheel
     - setuptools
   run:
     - python
-    - scikit-learn >=0.20.0
+    - scikit-learn >=1.0.0
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: should stay as it was before >= 0.20.0 ?
https://github.com/scikit-learn-contrib/category_encoders/blob/26ef26106fcbadb281c162b76258955f66f2c741/setup.py#L36

**Review by cbouss — COMMENTED** [review]
Have you tried re-enabling the skipped tests?

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml]: oh, must have been a mistake.

**Comment by boldorider4** [comment]
> Have you tried re-enabling the skipped tests?

I will try.

File: recipe/meta.yaml (hunk 1 above)
Comment (boldorider4) [reply recipe/meta.yaml]: fixed

**Comment by JeanChristopheMorinPerso** [comment]
@M-Waszkiewicz-Anaconda AFAIK this isn't needed anymore.

**Comment by M-Waszkiewicz-Anaconda** [comment]
> @M-Waszkiewicz-Anaconda AFAIK this isn't needed anymore.

it is not need for evalml - wrong refinement, but since  it on default channel we should still keep it up to date

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,38 +1,33 @@
...
     - pip
     - wheel
     - setuptools
   run:
     - python
-    - scikit-learn >=0.20.0
-    - scipy >=1.0.0
-    - pandas >=1.5.3
+    - scikit-learn >=1.0.0
+    - scipy >=0.20.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - scikit-learn >=0.20.0
    - scipy >=1.0.0
```

**Review by skupr-anaconda — APPROVED** [review]

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**